site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,413 @@
+
+With some reorganization of the code, we were able to reduce the size of the citra-room executable by 85%, decreasing citra’s compressed size by 6MB and uncompressed size by 43MB!
+Those who host rooms with citra-room will also no longer need to have OpenGL support on their machines.
+
+### UDS: implement GetApplicationData ([#5533](https://github.com/citra-emu/citra/pull/5533)) by [mnml](https://github.com/mnml)
+
+By implementing `GetApplicationData`, an issue is resolved in which multiplayer communications would only work one way in Dragon Ball Fusions, and fixes multiplayer altogether in Final Fantasy Explorers. `GetApplicationData` is used to load appdata from the current beacon, which can then only be used whilst connected to the network. 
+
+### service/frd: return cfg username on GetMyScreenName ([#5792](https://github.com/citra-emu/citra/pull/5792)) by [vitor-k](https://github.com/vitor-k)
+
+When playing multiplayer games, it is helpful to be able to identify your friends. But previously, when using the room feature in some games, everyone was named “Citra”!
+
+Obviously, this caused a lot of confusion and made it very hard to identify individual players. The problem was a stubbed function in the friend service. It was hardcoded to return “Citra” no matter the name the user had configured in the options. [vitor-k](https://github.com/vitor-k) corrected this and now the user-configured name is displayed correctly.
+
+{{< sidebyside "image" ""
+    "marioOld.png=Citra, Citra and... Citra?"
+    "marioNew.png=We're all here now! Let's play!" >}}
+
+# Frontend Improvements
+
+### Quick screenshot ([#6025](https://github.com/citra-emu/citra/pull/6025)) by [Z11-V](https://github.com/Z11-V)
+
+An arguably simpler way of taking screenshots in game, without taking you away from the action to name and save them! Use the Capture Screenshot hotkey to quickly save them to the screenshots folder in the [Citra User Directory](https://citra-emu.org/wiki/user-directory/).
+
+### Added Midnight Theme ([#6030](https://github.com/citra-emu/citra/pull/6030)) by [Daisouji](https://github.com/Daisouji)
+
+Sick of using just the default Light and Dark themes? Inspired by the yuzu theme of the same name, “Midnight Blue” and “Midnight Blue Colorful” are available in Citra to give you even more customizability than before!
+
+{{< figure src="midnight.png"
+    title="Ooh! Shiny new colors!" >}}
+
+### Citra SDL: graceful shutdown on application close ([#6102](https://github.com/citra-emu/citra/pull/6102)) by [ian-h-chamberlain](https://github.com/ian-h-chamberlain)
+
+When closing Citra SDL, the command line counterpart to our normal GUI version that most people will be used to, the window would remain open unintentionally which required a forceful shutdown to close properly. This would result in cleanup tasks, such as video dumping, failing. Which isn’t very nice if you’ve just spent a long time working on a video just to lose it! 
```

Comment:
Which isn't very nice -> This isn't very nice

Reply:
I disagree. Then you'd have two sentences starting with "This" referring to the same thing. Sounds worse imo

Reply:
I can't quote the academic reason why it is wrong, but it is wrong 😆 

Running the text through ChatGPT also agrees with me, and sounds a lot better.

> When exiting Citra SDL, the command-line equivalent of our standard GUI version that many users are accustomed to, the program window would sometimes remain open unintentionally. This issue could only be resolved by forcefully shutting down the application, which prevented important cleanup tasks like video dumping from completing successfully. This situation can be frustrating, especially if you've spent a significant amount of time working on a video project, only to lose it due to a technical glitch.